libp2p.nimble

```diff
@@ -56,14 +56,12 @@ task testpubsub, "Runs pubsub tests":
 
 task testfilter, "Run PKI filter test":
   runTest("testpkifilter")
-  runTest("testpkifilter", moreoptions = "-d:libp2p_pki_schemes=")
```

Comment:
why removed?

Reply:
As discussed with @richard-ramos, not specifying pki schemes will lead the test to use the default schemes, which is the same behavior of not specifying this flag anyways (i.e. lines 58 and 59 do the same thing)